modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/styles/components/_pagination.scss

```diff
@@ -0,0 +1,3 @@
+.page-item.active .page-link {
```

Comment:
Suggestion to verify inclusion in the "table.css" file

Reply:
Done!